automerge.yml

```diff
@@ -0,0 +1,28 @@
+name: automerge
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+  pull_request_review:
+    types:
+      - submitted
+  status: {}
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: automerge
+        uses: "pascalgn/automerge-action@de3a4d6a4975252444319cc9ab21b02d8d657229"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
According to the docs, a `github-actions` token will be used if one is not set.

Reply:
I _think_ the docs are saying `github-actions` will run the action if you don't set the `GITHUB_TOKEN`. The docs are confusing because they say `TOKEN` instead of `GITHUB_TOKEN`, but I think that's what they mean. It says: "To use this setting, you need to create a personal access token for the user"

And even if they run as `github-actions`, that user isn't going to have merge access, so this isn't going to work?